<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetModifyContact</class>
 <widget class="QWidget" name="WidgetModifyContact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifier Le Contact</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nom:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="Nom"/>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="checkName">
      <property name="text">
       <string>Selectionnez</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Prenom:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="Prenom"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Entreprise:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="Entreprise"/>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="CheckEntreprise">
      <property name="text">
       <string>Selectionnez</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Mail</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="Mail"/>
    </item>
    <item row="8" column="1">
     <widget class="QCheckBox" name="checkMail">
      <property name="text">
       <string>Selectionnez</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Telephone</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="Telephone"/>
    </item>
    <item row="10" column="1">
     <widget class="QCheckBox" name="checkPhone">
      <property name="text">
       <string>Selectionnez</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Photo</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QPushButton" name="Photo">
      <property name="text">
       <string>Choisir Photo</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QCheckBox" name="checkPhoto">
      <property name="text">
       <string>Selectionnez</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="checkPrenom">
      <property name="text">
       <string>Selectionnez</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="id"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Id Du Contact</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Modifier">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier Le Contact</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
